Source/WebCore/Modules/webcodecs/WebCodecsVideoDecoder.cpp

```diff
@@ -61,8 +60,34 @@ WebCodecsVideoDecoder::~WebCodecsVideoDecoder()
 {
 }
 
+static bool isValidDecoderConfig(const WebCodecsVideoDecoderConfig& config)
+{
+    // FIXME: Check codec more accurately.
+    if (!config.codec.startsWith("vp8"_s) && !config.codec.startsWith("vp09.00"_s))
```

Comment:
`config.codec.startsWith("vp09.00")` is really restrictive (see https://bugs.webkit.org/show_bug.cgi?id=241920).